packages/fx-core/src/plugins/solution/fx-solution/question.ts

```diff
@@ -36,29 +36,39 @@ export const BotOptionItem: OptionItem = {
   detail: getLocalizedString("core.BotOption.detail"),
 };
 
+export const BotNewUIOptionItem: OptionItem = {
+  id: "Bot",
```

Comment:
Will we still provide the legacy bot template since we have scenario-based command bot & notification bot?

Reply:
@dooriya checked with PM and bot template will be kept in options.